pgbot/commands/base.py

```diff
@@ -645,10 +659,20 @@ async def handle_cmd(self):
             )
         except discord.NotFound:
             # response message was deleted, send a new message
-            await embed_utils.send(
+            target_message = await embed_utils.send(
                 self.channel,
                 title=title,
                 description=msg,
                 color=0xFF0000,
                 footer_text=excname,
             )
+
+            task = asyncio.create_task(
+                message_delete_reaction_listener(
+                    target_message,
+                    self.author,
+                    "🗑",
+                    role_whitelist=common.ServerConstants.ADMIN_ROLES,
+                    timeout=30,
+                )
+            )
```

Comment:
Why is the task being re-created here if it is already created at the first few lines in this function call?

Reply:
I guess I should remove the `task =` assignment for clarity sake